test/config.sh

```diff
@@ -182,5 +198,5 @@ end_suite() {
     whitely assert_end
 }
 
-WEAVE=../weave
-DOCKER_NS=./bin/docker-ns
+WEAVE=$DIR/../weave
+DOCKER_NS=$DIR/../bin/docker-ns
```

Comment:
`weave_on` and `660_proxy_env_test.sh`. Could just use `weave` in both. After all, that's what `140_weave_local_test.sh` does.